src/wp-includes/taxonomy.php

```diff
@@ -4189,6 +4189,17 @@ function _update_post_term_count( $terms, $taxonomy ) {
 			$count += (int) $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(*) FROM $wpdb->term_relationships, $wpdb->posts WHERE $wpdb->posts.ID = $wpdb->term_relationships.object_id AND post_status IN ('" . implode( "', '", $post_statuses ) . "') AND post_type IN ('" . implode( "', '", $object_types ) . "') AND term_taxonomy_id = %d", $term ) );
 		}
 
+		/**
+		 * Fires when a term count is updated in the database.
```

Comment:
```suggestion
		 * Fires when a term count is calculated, before it is updated in the database.
```

Correct me if i missed anything.